engine/runtime/src/test/scala/org/enso/compiler/test/context/SuggestionBuilderTest.scala

```diff
@@ -908,10 +919,16 @@ class SuggestionBuilderTest extends AnyWordSpecLike with Matchers {
               externalId = None,
               module     = "Unnamed.Test",
               arguments = Seq(
-                Suggestion.Argument("a", "Number", false, false, None)
+                Suggestion.Argument(
+                  "that",
+                  "Standard.Base.Data.Numbers.Number",
+                  false,
+                  false,
+                  None
+                )
               ),
-              returnType    = "Unnamed.Test.MyType",
-              selfType      = "Number",
+              selfType      = "Standard.Base.Data.Numbers.Number",
+              returnType    = "Unnamed.Test.Foo",
```

Comment:
Yes, that seems correct. `selfType` is now FQN. `returnType` is the type we convert to.